python/cudnn/gemm/cutedsl/grouped/unfused/api.py

```
@@ -234,9 +259,9 @@ def grouped_gemm_wrapper_sm100(
     b_dtype: Optional[torch.dtype] = None,
     b_major: str = "k",
     prob_tensor: Optional[torch.Tensor] = None,
-    acc_dtype: torch.dtype = torch.float32,
-    c_dtype: torch.dtype = torch.bfloat16,
-    d_dtype: torch.dtype = torch.bfloat16,
+    acc_dtype: Optional[torch.dtype] = None,
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)